utils/doclint/check_public_api/index.js

```diff
@@ -807,6 +814,16 @@ function compareDocumentations(actual, expected) {
      * as they will likely be considered "wrong" by DocLint too.
      */
     if (namingMismatchIsExpected) return;
+
+    /* Some methods cause errors in the property checks for an unknown reason
+     * so we support a list of methods whose parameters are not checked.
+     */
+    const skipPropertyChecksOnMethods = new Set([
+      'Method Page.deleteCookie() ...cookies',
+      'Method Page.setCookie() ...cookies',
+    ]);
+    if (skipPropertyChecksOnMethods.has(source)) return;
```

Comment:
I have no idea why this change suddenly causes this; I've manually checked that the new types are identical to the old types in all but name.

At this point the doclint tooling is a mystery to me and I'd rather hack around it as we're quite close to removing it.

Reply:
ack, sg